docs/commands/csv2po.rst

```diff
@@ -88,34 +92,45 @@ The resultant CSV file has the following layout
 |  C     | Target Language | The msgstr or target language               |
 +--------+-----------------+---------------------------------------------+
 
+
 .. _csv2po#examples:
 
 Examples
 ========
 
-These examples demonstrate the use of csv2po::
+These examples demonstrate the use of csv2po:
+
+.. code-block:: console
 
   po2csv -P pot csv
 
-We use the :opt:`-P` option to recognise POT files found in *pot* and convert
-them to CSV files placed in *csv*::
+
+We use the :opt:`-P` option to recognise POT files found in :file:`pot` and
+convert them to CSV files placed in :file:`csv`:
```

Comment:
Ah true.  But my central point is readability. I have to now parse that string more slowly and carefully because of those markers.  So I don't agree that they are adding any value.